src/main/resources/application.yml

```
@@ -6,11 +6,13 @@ spring:
     password: ${DATABASE_PASSWORD}
 
   jpa:
+    show-sql: true
     hibernate:
       ddl-auto: update
     properties:
       hibernate:
-        auto_quote_keyword: true
+        format_sql: true
+        show_sql: true
```

```suggestion
        format_sql: true
        show_sql: true
        default_batch_fetch_size: 500
```
[P2] jpa properties에 이 옵션 추가하는 것은 어떤가요?
OneToMany로 연관된 엔티티들을 batch 단위로 조회해서 DB connection을 줄일 수 있습니다!

---

헐 이 옵션은 처음보는데, OneToMany로 연관된 엔티티가 많을 저희 서비스에 넘 좋을거같아요! 바로 반영~